winlogbeat/eventlog/wineventlog.go

```diff
@@ -197,37 +186,10 @@ func (l *winEventLog) openChannel(bookmark win.EvtHandle) error {
 func (l *winEventLog) openFile(state checkpoint.EventLogState, bookmark win.EvtHandle) error {
 	path := l.channelName
 
-	h, err := win.EvtQuery(0, path, "", win.EvtQueryFilePath|win.EvtQueryForwardDirection)
+	h, err := win.EvtQuery(0, path, l.query, win.EvtQueryChannelPath|win.EvtQueryReverseDirection)
```

Comment:
As this is a part of the functionality that we want to add and as this is not a part of what was added by Continuum approach, I propose to revert it and add as separate PR

Reply:
I kept it considering this will not impact the existing flow for event log plugin but then it would be better to revert code changes for openFile method and let it be part of another PR.

Reply:
Reverted openFile method .